packages/@aws-cdk/aws-apigateway/lib/stage.ts

```diff
@@ -348,4 +350,93 @@ export class Stage extends Resource implements IStage {
       };
     }
   }
+
+  /**
+   * Returns the given named metric for this stage
+   */
+  public metric(metricName: string, props?: cloudwatch.MetricOptions): cloudwatch.Metric {
+    return new cloudwatch.Metric({
+      namespace: 'AWS/ApiGateway',
+      metricName,
+      dimensionsMap: { ApiName: this.restApi.restApiName, Stage: this.stageName },
+      ...props,
+    }).attachTo(this);
+  }
+
+  /**
+   * Metric for the number of client-side errors captured in a given period.
+   *
+   * Default: sum over 5 minutes
```

Comment:
throughout this PR, default comments should read analogously to:

```suggestion
   * @default - sum over 5 minutes
```